imtascii.py

```diff
@@ -1,11 +1,20 @@
 from PIL import Image
 from time import sleep, time
 
+import argparse
+
```

Comment:
Consider add some comments to explain the code.

Reply:
I dont think this is necessary, but thanks so much for the suggestion